src/styles/main.scss

```
@@ -1,3 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+$day-side: 100px;
+$day-size: 30px;
+$days: 0 "mon",
+       1px "tue",
+       2px "wed",
+       3px "thu",
+       4px "fri",
+       5px "sat",
+       6px "sun";
+
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  @each $i, $day in $days {
+    &--start-day--#{$day} {
+      .calendar__day:nth-child(1) {
+        margin-left: $i * 100 + $i;
+      }
+    }
+  }
+
+  @for $day from 28 through 31 {
+    &--month-length--#{$day} {
+      @for $i from $day through 31 {
+        .calendar__day:nth-child(#{$i + 1}) {
+          display: none;
+          opacity: 0;
+        }
+      }
+    }  
+  }
+
+
+&__container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $day-side * 7 + 10px;
+}
+
+&__day {
```

since there are nested selectors they should have an indent
```suggestion
  &__container {
    display: flex;
    flex-wrap: wrap;
    gap: 1px;
    width: $day-side * 7 + 10px;
  }
  
  &__day {
```

---

take a look for indents please